topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -243,6 +251,94 @@ def _decl_spatial_pack(cfg, data, kernel, strides, padding, dilation, layout, ou
                          name='output_unpack', tag='spatial_conv2d_output')
     return output
 
+
+def _decl_spatial_pack_nhwc(cfg, data, kernel, strides, padding, dilation,
+                            layout, out_dtype, num_tile):
+    assert layout == "NHWC", "Only support NHWC"
+    # create workload according to raw arguments
+    out_dtype = out_dtype or data.dtype
+    N, IH, IW, CI = get_const_tuple(data.shape)
+
+    # TODO? Dilation
```

Comment:
insert assertion for dilation

Reply:
Check now added to confirm dilation is either 1 or (1, 1)